scrunch/datasets.py

```diff
@@ -2495,3 +2493,16 @@ def transformations(self):
         if 'transform' in self.view:
             return self.view.transform.insertions
         return None
+
+    def rollup_resolution(self, resolution):
+        """
+        PATCHes the rollup_resolution attribute of a datetime variable. This is the
+        equivalent to the UI's change resolution action.
+        """
+        assert self.type == 'datetime', 'Method only allowed for datetime variables'
+        self.dataset._validate_vartypes(self.type, resolution=resolution)
+
+        payload = json.dumps({'rollup_resolution': resolution})
+        resp = self.resource.patch(payload)
+        self.dataset._reload_variables()
+        return self.dataset[self.alias]
```

Comment:
If you use the `.edit()` pycrunch method, then the resource's attribute will get automatically updated so no need to re-read the Variable instance or to do a full reload of all the variables (can be a expensive GET). You could just return `self`.

Reply:
👍